StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1009ClosingParenthesisMustBeSpacedCorrectly.cs

```diff
@@ -172,12 +183,18 @@ private static void HandleCloseParenToken(SyntaxTreeAnalysisContext context, Syn
                 if (!precedesStickyCharacter && !followedBySpace && !lastInLine)
                 {
                     // Closing parenthesis must{} be {followed} by a space.
-                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, token.GetLocation(), string.Empty, "followed"));
+                    var properties = ImmutableDictionary.Create<string, string>()
```

Comment:
Why we create `ImmutableDictionary` and then mutate it?
`ToImmutableDictionary` extension also accepts `IEnumerable<KeyValuePair<,>` which implemented by `Dictionary<,>`